app/assets/javascripts/hello-dev.js

```diff
@@ -18,5 +18,5 @@ var bigDev =
                                                      \n';
 console.log(
   bigDev,
-  "Hey there! Interested in the code behind dev.to? Well you're in luck - we're open source! Come say hi, tell us what you're debugging, or even lend a hand in our repo - https://github.com/thepracticaldev/dev.to/"
+  "Hey there! Interested in the code behind dev.to? Well you're in luck - we're open source! Come say hi, tell us what you're debugging, or even lend a hand in our repo - https://github.com/thepracticaldev/dev.to/ \n \Did you find a bug or vulnerability? Check out our bug bounty info here: https://dev.to/security"
```

Comment:
Just a suggestion, wouldn't it look cleaner like this?

```suggestion
  "Hey there! Interested in the code behind dev.to? Well you're in luck - we're open source! Come say hi, tell us what you're debugging, or even lend a hand in our repo - https://github.com/thepracticaldev/dev.to/ \n",
  "Did you find a bug or vulnerability? Check out our bug bounty info here: https://dev.to/security"
```